index.html

```diff
@@ -0,0 +1,460 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <h1 id="logo">Aziz Mejri</h1>
+      <button type="button" class="menu-bars">
+        <i class="fas fa-bars"></i>
+      </button>
+      <div class="dropdown-menu hide">
+        <div class="mobile-menu">
+          <button class="close">&times;</button>
+        </div>
+        <ul class="mobile-menu-options">
+          <li class="dropdown-option"><a href="#projects">Portfolio</a></li>
+          <li class="dropdown-option"><a href="#aboutme">About</a></li>
+          <li class="dropdown-option"><a href="#contact">Contact</a></li>
+        </ul>
+        <div class="dropdown-bottom">
+          <div class="dropdown-divline"></div>
+        </div>
+      </div>
+      <nav id="desktop-nav">
+        <ul>
+          <li><a class="nav-link" href="#projects">Portoflio</a></li>
+          <li><a class="nav-link" href="#aboutme">About</a></li>
```

Comment:
Please, make sure that the `href` attribute of your links points to a proper and intended section of your page (or external link) to avoid a malfunction of the site and hampering the user experience.

The _aboutme_ referenced on line 33 above is different from the name of the section _about-me_ given on line 263. Check the image below:

![image](https://user-images.githubusercontent.com/20554583/184082319-62f7d2e4-8e2b-464c-ae2d-642b2fcbefd4.png)